packages/snap-preact/components/src/components/Molecules/FacetHierarchyOptions/FacetHierarchyOptions.tsx

```diff
@@ -37,7 +38,7 @@ const defaultStyles: StyleScript<FacetHierarchyOptionsProps> = ({ theme, horizon
 				},
 				'&.ss__facet-hierarchy-options__option--return': {
 					'&:before': {
-						content: `'\\0000ab'`,
+						content: `${returnIcon ? '""' : "'\\0000ab'"}`,
 						padding: '0 2px 0 0',
 						color: theme?.variables?.colors?.primary,
```

Comment:
I think it may be preferable to just not generate this style at all when `returnIcon` is specified. However... it seems this padding is needed to give a bit of alignment.